code/game/objects/items/devices/defibrillator.dm

```diff
@@ -77,11 +77,11 @@
 	ready = !ready
 	user.visible_message(SPAN_NOTICE("[user] turns [src] [ready? "on and takes the paddles out" : "off and puts the paddles back in"]."),
 	SPAN_NOTICE("You turn [src] [ready? "on and take the paddles out" : "off and put the paddles back in"]."))
+	playsound(get_turf(src), "sparks", 25, 1, 0)
 	if(ready)
-		w_class = SIZE_LARGE
```

Comment:
Cough Cough think this reverses paddles change, please add it to changelog if you really want it in this (personally don't think it's related to sound)